linera-rpc/src/grpc/client.rs

```diff
@@ -323,7 +323,7 @@ impl ValidatorNode for GrpcClient {
         Ok(self
             .client
             .clone()
-            .blob_last_used_by(<BlobId as Into<api::BlobId>>::into(blob_id))
+            .blob_last_used_by(<BlobId as TryInto<api::BlobId>>::try_into(blob_id)?)
```

Comment:
Is that annotation needed?
Would `blob_id.try_into()?` or `api::BlobId::try_from(blob_id)?` also work?

Reply:
The latter worked!